src/auto/activate_action.rs

```diff
@@ -0,0 +1,29 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+use ShortcutAction;
+
+glib_wrapper! {
+    pub struct ActivateAction(Object<gtk_sys::GtkActivateAction, gtk_sys::GtkActivateActionClass, ActivateActionClass>) @extends ShortcutAction;
+
+    match fn {
+        get_type => || gtk_sys::gtk_activate_action_get_type(),
+    }
+}
+
+impl ActivateAction {
+    pub fn get() -> Option<ShortcutAction> {
```

Comment:
This needs a `type Gtk.ActivateAction` annotation (and same for the others)

Reply:
Reported upstream as part of the broken annotations issue 
https://gitlab.gnome.org/GNOME/gtk/-/issues/3281